tests/unit/percentage-step-form.spec.ts

```diff
@@ -91,7 +91,7 @@ describe('Percentage Step Form', () => {
         keyword: err.keyword,
         dataPath: err.dataPath,
       }));
-      expect(errors).to.eql([{ keyword: 'nameAlreadyUsed', dataPath: '.new_column' }]);
+      expect(errors).to.eql([{ keyword: 'not', dataPath: '.new_column' }]);
```

Comment:
I like the approach but now we've got an error message less clear than before :/

Reply:
Indeed, that's something I would need to dig a bit more but I'm not sure this is something we can change or configure.